docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,359 @@
+
+Finally connecting your local data is made easier! Welcome drag & drop in Grafana!
+The drag & drop functionality allows users to connect their csv, excel or numbers files by simply dragging and dropping them on a dashboard, or even into the editor directly.
+
+Once dropped a table will be created by default containing the data that is living in the file itself.
+
+By default, the drag & drop feature is not available out of the box. In order to utilize this functionality panel you will need to enable the `featuretogglename` feature toggle.
+
+As of now drag & drop supports the following scenarios in G10:
+
+- Drag & drop files on an empty dashboard
+- Drag & drop files into the panel editor
+- Replace files in the panel editor
+- Default table panel creation
+
+As of now the data is being stored on the dashboard json and has a 1MB size limit. Obviously it’s our mission to increase the limit without decreasing the performance in the near future.
+To learn more about drag & drop, please refer to the official documentation.
+
+The list of requests is already growing and of course we will look into those. So stay tuned for more features coming in the near future that will give an even better experience using the the drag & drop datasource.
+
+### Canvas GA
+
+<!-- Nathan Marrs -->
+
+We are promoting the canvas panel out of beta and into general availability. Over the past several months we have introduced several enhancements to the panel such as the ability to draw connections between elements, the ability to set the color and size of connections based on data, and the ability to add data links. We are excited to include canvas as a first class citizen in Grafana’s core panel library. To learn more about the Canvas panel, refer to [Canvas]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-canvas-service-graph.png" max-width="750px" caption="Canvas service graph" >}}
+
+### New Trend Panel
+
+<!-- Nathan Marrs -->
+
+_Available in **beta** in all editions of Grafana_
+
+We are excited to introduce a new panel that allows you to display trends where the x axis is not time. This new beta panel addresses gaps that were not solved by either the Time series or XY Chart panels.
+
+### Time series time region support
+
+<!-- Nathan Marrs -->
+
+We have implemented support for adding time regions to the time series panel. Time regions provide a more contextualized experience, enabling you to highlight certain days of the week, such as Monday-Friday to display work weeks right alongside your data. Time regions are a useful way to highlight specific parts of a day like night, work hours, or whatever you define for each day. They allow the viewer to quickly orient themselves in parts of the day and/or ignore highlighted parts of the time series.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-0-time-regions.png" max-width="750px" caption="Time regions" >}}
```

Comment:
@nmarrs We could very shortly say where to find this in the configuration of a time series panel, or link to a page to learn more how to set this up for a panel. I took a while to try to find it in our 10.x test instance and I couldn't find it for a timeseries panel.

Reply:
I agree on this. Time regions are set up through dashboard annotations instead of a specific panel editor option

Reply:
done 👍 [commit here](https://github.com/grafana/grafana/pull/64927/commits/bdbd773d78414759c4540e230f29ce589e1f31a7)